stylesheets/stylesheet.css

```diff
@@ -704,7 +704,7 @@ form {
   text-align: center;
   padding: .5em 1em;
   margin: 0 auto 0;
-  right: 2em;
+  right: 15em;
```

Comment:
This breaks viewing the header when the viewport is narrow, because it's a hard-coded value:

<img width="1016" alt="Screenshot 2023-08-27 at 4 29 35 PM" src="https://github.com/up-for-grabs/up-for-grabs.net/assets/359239/c658d33c-3a5d-4aa8-982c-6b322c8fe9d0">

I'd recommend looking for a solution that works better at different widths, and considering how the header looks when testing in responsive mode in your local browser dev tools:

<img width="603" alt="Screenshot 2023-08-27 at 4 27 58 PM" src="https://github.com/up-for-grabs/up-for-grabs.net/assets/359239/f7f724b1-2422-4ff1-b09c-8ba5655adb3c">